i18n/en_US.json

```diff
@@ -187,7 +187,7 @@
 				"saveAwardButton": "Save award",
 				"removeAwardButton": "Remove award",
 				"removeAwardTitle": "Remove award",
-				"removeAwardText": "Are you sure you want to remove this wawrd?",
```

Comment:
:smile:

Reply:
Oh, I thought I somehow did it wrong again! :P